newIDE/app/.gitignore

```diff
@@ -16,7 +16,6 @@ build
 
 # misc
 .DS_Store
-.env
```

Comment:
If we want to leak the keys why not, but I think it's unexpected.

Reply:
@4ian told me that this file is not used anymore, if you have one, I think you can delete it.

Reply:
Oh ok sorry

Reply:
Yep it's gone for long! :)